pkg/kube/object.go

```diff
@@ -177,6 +179,15 @@ func (ci ContainerImages) FromJSON(value string) error {
 	return json.Unmarshal([]byte(value), &ci)
 }
 
+func (ci ContainerImages) DeepEqualTo(m map[string]bool) bool {
```

Comment:
The name of this func is not good, and the internals can probably also be improved. But I suggest to move the func here, with `ContainerImages` as a receiver.